### ISSUE TITLE
White List Entry

### ISSUE THREAD

**Issue by cutic** [Thaerious/NERVE#75]
Once you select it, it remains selected permanently (whether you open CTRL-L , or Link or any dialogue window that contains White List Entry option). Not sure if we want this or not?

**Comment by Thaerious** [comment]
Yes that is desired behavior as.  If we want a one-off whitelist entry then a checkbox wouldn't be the ui component of choice.
